app/controllers/volunteer_ops_controller.rb

```diff
@@ -78,27 +78,34 @@ def authorize
     # set @organisation
     # then can make condition:
     # unless current_user.can_edit? organisation
-    unless org_owner? || superadmin?
+
```

Comment:
can_edit? contains the check:

  # can create or edit an organization
  def can_edit? org
    superadmin? || (!org.nil? && organisation == org)
  end